book/types_of_data.md

```diff
@@ -245,6 +245,21 @@ However, if a record has a key name that can't be expressed as a bare string, or
 false
 ```
 
+To make a copy of a record with new fields, you can use the spread operator (`...`):
+
+```nu
```

Comment:
i think it should be `nushell` for the syntax highlighting to kick in, or do we support `nu`?

Reply:
The other code blocks in the same file used `nu`, so I think both work

Reply:
okey, gotcha :ok_hand: